floppyforms/fields.py

```diff
@@ -7,16 +8,20 @@
                       EmailInput, NullBooleanSelect, SlugInput, IPAddressInput,
                       SplitDateTimeWidget, SplitHiddenDateTimeWidget,
                       MultipleHiddenInput)
+if django.VERSION < (1, 9):
+    from .widgets import IPAddressInput
 
 __all__ = (
     'Field', 'CharField', 'IntegerField', 'DateField', 'TimeField',
     'DateTimeField', 'EmailField', 'FileField', 'ImageField', 'URLField',
     'BooleanField', 'NullBooleanField', 'ChoiceField', 'MultipleChoiceField',
-    'FloatField', 'DecimalField', 'SlugField', 'RegexField', 'IPAddressField',
+    'FloatField', 'DecimalField', 'SlugField', 'RegexField',
     'GenericIPAddressField', 'TypedChoiceField', 'FilePathField',
     'TypedMultipleChoiceField', 'ComboField', 'MultiValueField',
     'SplitDateTimeField',
 )
+if django.VERSION < (1, 9):
```

Comment:
I would move this down to the `class IPAddressField` definition to keep the logic for the field together. Otherwise we might forget removing statement once we drop Django 1.8 support.